### ISSUE TITLE
New Icon for IINA

### ISSUE THREAD

**Issue by bequadro** [lhc70000/iina#1782]
I talked to @lhc70000 about a new icon for IINA v1.0, so I'm opening an issue here for discussion.

First of all, the new icon should adhere to the macOS system icon standards as described in [here](http://martiancraft.com/blog/2014/07/inspecting-yosemite-icons/).

So here's my entry and the rationale behind it:

<img width="670" alt="icon" src="https://user-images.githubusercontent.com/30233933/41204067-053e5828-6ce0-11e8-8799-d05366dec472.png">

I like the name IINA, so I played around with the letters, mainly I and A, which mimic a traditional play/pause logo. The chromatic aberration effect is achieved by stacking red, green and blue layers (like RGB pixels in a screen). This can only really be seen when the icon is large, so IINA keeps an understated look in the dock.

<img width="628" alt="screenshot" src="https://user-images.githubusercontent.com/30233933/41203992-2fed0f16-6cdf-11e8-8487-79c29bda4983.png">

I'm looking forward to your feedback or your own suggestions!

**Comment by jerkstore369** [comment]
I think the current icon looks fine. I like the black/blue over black/white.

**Comment by bequadro** [comment]
@inflation that's kind of the intended effect!
@jerkstore369 here are some previous, more colorful variations of the icon.

![icon 3 19](https://user-images.githubusercontent.com/30233933/41241530-76d754a4-6d9d-11e8-9beb-fc3a57efd09f.png)
![icon 3 20](https://user-images.githubusercontent.com/30233933/41241532-77045d28-6d9d-11e8-8e4d-b1beeb950c50.png)
![icon 3 21](https://user-images.githubusercontent.com/30233933/41241533-7732c028-6d9d-11e8-8032-d2f7242d93f5.png)

**Comment by kxxc** [comment]
Great! maybe make the inside II> colors looks like the iTunes APP?

**Comment by fabi94music** [comment]
I like the white one

**Comment by ColeTownsend** [comment]
For the sake of simplicity, I’d probably go with the all white one - but I think the abberation CMYK effect on the original is a cool nod to early AV technology. The dark one also goes well with the dark OS changes. 

I’d say the strongest iterations are the white and original, and we can leave that choice up to the makers here?

**Comment by jerkstore369** [comment]
Not crazy about the white one. Too much white as it is. Not the greatest to those who like dark mode. The first new one (black/blue) is not bad. Just maybe make the outer ring the same colour as the white one and make the blue brighter, higher contrast. That should be good.

**Comment by kxxc** [comment]
I like the blue one, more easier to find out in the dick

**Comment by saagarjha** [comment]
I'd fix that typo if I were you…

**Comment by ColeTownsend** [comment]
Oh knowing what MINA looks like – I'd push forward with the dark direction then.

**Comment by RuiCarneiro** [comment]
I think the white one looks too much Mavericks-era, it won't age well surely.

The middle one is the best one yet.

**Comment by bequadro** [comment]
@ErikMinekus Think of it as an easter egg. It's only visible in full size, which you'll basically never see and it still looks pretty clean/minimalist in the dock.

Nevertheless, I came across this MUSIC banner recently: [https://www.apple.com/v/music/g/images/shared/og_image_family.png](https://www.apple.com/v/music/g/images/shared/og_image_family.png)

I tried recreating a similar 3D effect, albeit a little less glossy and saturated. Have a look.

<img width="670" alt="icon 3 22" src="https://user-images.githubusercontent.com/30233933/41898724-02f03454-792b-11e8-8931-3869b511859d.png">
<img width="670" alt="icon 3 21" src="https://user-images.githubusercontent.com/30233933/41898738-0a1e4d9c-792b-11e8-97c1-0b58def21eb4.png">
<img width="670" alt="icon 3 19" src="https://user-images.githubusercontent.com/30233933/41898737-09fac9d0-792b-11e8-95e4-0adf8c17079c.png">

**Comment by fabi94music** [comment]
WOW! The Green/Blue one looks stunning!!

**Comment by euf** [comment]
Is it possible to combine the last green-blue icon with a 3D effect and the chromatic aberration effect from the very first icon?

**Comment by ErikMinekus** [comment]
@bequadro I can't really find any macOS system or app icons that use such overstated glow effects or shadows. The iTunes 12 icon for example only has a small inner shadow on the top and left of the music note. I don't mind the outer glow though.

I still prefer the second icon, but you could put an outer glow on it to smoothen the transition from blue/green to black. Or if you want an inner shadow, I think the fourth icon but with a black background and less pronounced inner shadows would look nice too.

**Comment by POPOEVER** [comment]
Thanks for the work but sorry, none of them looks appealing to me, still love the current one, it's clear, sharp and self-explaining, and pretty standing out from other icons

**Comment by ultratiem** [comment]
Speaking as a designer, you're wasting your time. People here aren't qualified to give you anything more than a "I like it" or "I hate it" reply, which isn't all that useful to you. You're never going to make everyone happy and it's not a smart move to base your designs on fickle taste from people who may or may not even be in your field. You can glean general points to help you better tailor the emotionality of the design but that's pretty much the extent of if (e.g., "this design makes me feel like I'm in a hospital" isn't ideally what you're after if you're designing a new health product). You're better off discussing this with lhc7 and coming up with something you both like. People can deal with it.

With that said, I challenge you to think outside of the standard "default" icon template Apple generally uses. You can of course adhere to it, but there's no hard and fast rule there. Look to Transmit 5, Sketch, or Xscope for inspiration. Don't limit yourself to a boring old template unless you feel it best fits the app. I mean, Apple didn't think so with regards to QuickTime. Be creative.

PS: I very much like the spartan white design even though it does come across like the installer for macOS. Still cool 👍

**Comment by ultratiem** [comment]
Thanks @bequadro. A retro TV set would be neat. I do very much like the RGB effect but I can see why people felt it was too artistic. This is an open source project so anyone is free to fly with whatever, and I do love vintage art, but I think something more modern and simple would be ideal. With that said, as an artist, have fun. If you love the first iteration, I encourage you to refine it. Perhaps increase the RGB effect (pull the layers farther apart) so they look intentional and work better at lower resolutions. Balance each unique size accordingly rather than just shrinking the icon down. That way it'll come thru at every size. Perhaps try adding some scanlines. I'm a bit wary to suggest this as they only tend to look good at a 1:1 scale, and that's not going to happen outside of Launchpad (which uses a standard size) as every other icon will likely be off-scale and set to whatever size by the user (such as Finder and the Dock). But hey, give it a shot! Looking forward to what you come up with :)

**Comment by LeoNatan** [comment]
Also consider having two variants for Mojave: one for dark mode and one for light mode.

**Comment by ykqmain** [comment]
I prefer the F8 on the Apple Keyboard, like ⏯

**Comment by jerkstore369** [comment]
Aren't qualified? It's an icon, there are certain objectively obvious things in icons that can make them appealing. My previous suggestions, what I was getting at, was to sort of make the look more consistent with the OS with brighter colours, higher contrast, brighter outer ring. Absolutely nothing wrong with that. But I'm not qualified to make that suggestion? You designers are something else. Half the business logos out there are ridiculous.

**Comment by Wayang2019** [comment]
dark ... 
<img width="670" alt="iina logo dark" src="https://user-images.githubusercontent.com/42605455/46566531-9f2ba280-c94a-11e8-97a6-648f2ede275d.png">

and light ...
<img width="670" alt="iina logo light" src="https://user-images.githubusercontent.com/42605455/46566532-9f2ba280-c94a-11e8-8e12-1b7c550ed8f1.png">

**Comment by saagarjha** [comment]
@LeoNatan I'm not sure if this is something we should do. Apple's unofficial position on this seems to be "make an icon that works for both dark and light mode", given that they haven't provided alternate icons anywhere.

**Comment by LeoNatan** [comment]
@saagarjha Agree. Even achieving what they have done with folder and trash icons is not possible with current asset catalogue options for app icons.

**Comment by inazarab** [comment]
The only thing I don't like in the logo is where it overlaps in the arrow so the corner is more opaque. I think it'd look sleeker if it was the same opaqueness throughout. Otherwise I think its awesome!

**Comment by saagarjha** [comment]
@LeoNatan It's possible, but not with asset catalogs. You'd need to write a dock tile plugin that responds to the system appearance.

**Comment by LeoNatan** [comment]
Isn’t that just for the Dock? If I want to have different document icons, that would not be possible.

**Comment by schrodincat** [comment]
I think you need to make chromatic aberration effect stronger, there is a little point in it, if it's not seen in Dock

**Comment by mePy2** [comment]
I searched for open issues about the icon and I found this one. I didn’t read all the comments etc. but I only would like to say that personally I don’t like the current black/white/blueish icon. And with “I don’t like” I mean it’s not 100% macOS (it’s not about personal taste).

The first icon above it’s better IMO, but I didn’t make any try, I’m a bit in a hurry now.
Good job to everyone though :)

**Comment by bequadro** [comment]
So I took some time to review the icon again. I ditched the logo icon and reused its best parts in a new concept.

The new icon uses the default iOS icon shape — a squircle — stretched out to resemble an old TV set, which in turn shows the _zap_ you see when turning off a CRT monitor ([as seen here](https://youtu.be/6hKIHF5cULg)).

I made two versions, one with a retro looking green glow and one with the colors from the chromatic aberration above (shoutout to Felipe Pantone).

![crt zap](https://user-images.githubusercontent.com/30233933/53124904-e82c6e00-355c-11e9-93df-6ee639d64c1b.png)
![crt colors](https://user-images.githubusercontent.com/30233933/53124903-e82c6e00-355c-11e9-8fcf-3a881fd80eea.png)


I also adapted the original logo into a new logotype for IINA, in case there is any need for that.
![logotype](https://user-images.githubusercontent.com/30233933/53124918-efec1280-355c-11e9-818a-c22971b7ff1e.png)
Looking forward to your feedback and critiques!

**Comment by inazarab** [comment]
I like them both a lot. My preference is the chromatic aberration version, but I love them both. I think the chromatic aberration would look cooler if there were a bit more aberration. If you could push that effect more, I think it would stand out. They are both awesome though.

**Comment by LeoNatan** [comment]
I don't like this design. It doesn't look anything like a macOS icon.

**Comment by inazarab** [comment]
I'm not sure what you mean by "It doesn't look anything like a macOS icon" I think it fits in with all the other icons on my dock. There's not really a standard design: The message app is a bubble icon, safari is a compass, CleanMyMac is a computer monitor, and Microsoft office is a book. MediaInfo is three colored lines, a question mark, and three soundwaves. Some of the icons have a flat look, and some have a more 3D look. Some have outlines and some don't. Some are neon colors some are pastel. There is no standard design language I can find that it doesn't fit in with.

**Comment by inazarab** [comment]
@LeoNatan I'm not sure what you mean by "It doesn't look anything like a macOS icon" I think it fits in with all the other icons on my dock. There's not really a standard design: The message app is a bubble icon, safari is a compass, CleanMyMac is a computer monitor, and Microsoft office is a book. MediaInfo is three colored lines, a question mark, and three soundwaves. Some of the icons have a flat look, and some have a more 3D look. Some have outlines and some don't. Some are neon colors some are pastel. There is no standard design language I can find that it doesn't fit in with.

**Comment by bequadro** [comment]
@LeoNatan It blends pretty nicely into the dock. The border matches Safari, Books, etc.

<img width="491" alt="dock" src="https://user-images.githubusercontent.com/30233933/53129285-79a0dd80-3567-11e9-83a6-507a73c8476d.png">

**Comment by LeoNatan** [comment]
http://martiancraft.com/blog/2014/07/inspecting-yosemite-icons/
https://developer.apple.com/design/human-interface-guidelines/macos/icons-and-images/app-icon/

**Comment by bequadro** [comment]
@LeoNatan I'm aware of link #1 (see my first post). And to cite form link #2:

> Consider giving your app icon a realistic, unique shape. In macOS, app icons can have the shape of the objects they depict. A unique outline focuses attention on the object and makes it easy to recognize the icon at a glance.

**Comment by LeoNatan** [comment]
Yes. The Safari icon remains the theme and still manages to be unique. So are the other icons (Chrome excluded, it's a multiplatform crapola). I think you misunderstand what apple are saying. In your own words: "The new icon uses the **default iOS icon shape**"; uhmm why? This is not iOS.

**Comment by inazarab** [comment]
@LeoNatan The guidelines are up for personal interpretation they don't realy give any hard rules. so what you might interpret as not working others feel differently. A guideline is just that a guideline. t is meant to guide. It's not a list of checkmarks that you have to have to check every box. When you say, "I think you misunderstand what apple are saying." I'm not sure how anyone but Apple knows what Apple is saying. We all interpret things differently, neither of us is right or wrong it's just personal taste.

**Comment by bequadro** [comment]
@LeoNatan Well the shape is primarily that of an old TV set. The fact that it uses a stretched squircle is just a neat tidbit, nothing more.

![tv](https://user-images.githubusercontent.com/30233933/53131186-5cbad900-356c-11e9-951a-a5a915a3fefa.jpg)

**Comment by bequadro** [comment]
I added scan lines to the icons for additional texture and retro feel. Lmk what you think.

<img width="687" alt="comparison" src="https://user-images.githubusercontent.com/30233933/53185095-9f2cf600-35fe-11e9-9ae8-b5541796053f.png">

**Comment by ultratiem** [comment]
@bequadro I like that you're experimenting. I just want to point out that IINA plays everything, not just movies. Music too. If you're determined to go for an old CRT (I love vintage tech), try to tease out what elements sell CRT. I'm afraid much of it is lost in your icon and you've just focused on wedging in a single (and arguably subtle element) of the display. This will be lost on people. If you had to lose your brand for this, is it worth it?

I think you did a great job with the scan line flicker, but then just went all flat/simple, giving it a half in, half out feel.

Keep exploring and trying to really isolate the key features that make something, something. And don't be afraid to go a bit skeuomorphic. There are plenty of icons in macOS that still adopt this style (Keychain, Automator, Disk Utility, even Mail).

**Comment by bequadro** [comment]
@ultratiem My idea for this icon was a box that can play anything. So I tried to stay somewhat abstract to also include the audio aspect of IINA. That's why I like the colorful icon more because the line can be interpreted as a progress bar.

What are your suggestions for additional skeuomorphism? 3D? Reflections? Knobs?

**Comment by whatsnewsisyphus** [comment]
I like the Colors scan without the 3d, the reflection you used is that of a matte surface so the skeumorphism is off looking, as in, a glass CRT would gloss very differently. The border also starts being a little too prominent. Also, the color spectrum seems a bit random, try a RGBW variant maybe? I liked the first one best with the play pause I also use IINA for audio a lot and so the TV variant is not fully encompassing but since we are riffing about here. Could you try that with a rectangular background instead of a circle?

Some inspiration for tvs turning off (http://www.stephantillmans.com/luminant-point-arrays)

I'm not opposed to logo variants though it certainly is not the fashion these days
![screenshot 2019-03-01 17 13 27](https://user-images.githubusercontent.com/454772/53669538-61a31b00-3c45-11e9-80bb-19ec3c45fea2.png)

**Comment by intrigua** [comment]
It's matter of taste but current IINA icon makes me think it was ordered on Fiverr for the lowest price tier, not quite what I expect to have in such a great media player.

**Comment by whatsnewsisyphus** [comment]
Is it supposed to be いいね ? Or does iina come from somewhere else?

**Comment by jerkstore369** [comment]
Jesus. There's nothing wrong with the icon. Ive seen much worse. It's a circle with play inside, what more do you want? Who cares? I think it should be circle like most other new macOS icons, now you people are trying to make it a different ridiculous shape. Why don't you just make your own icon in photoshop and and paste it in file > get info and there, you can have your own nice looking icon. Problem solved. Closed.

**Comment by mcc108** [comment]
When will the logo be replaced? The current icon is really ugly.

**Comment by lemikeone** [comment]
The white one is so cool, thanks @chrenn !

<img width="1440" alt="2020-04-03_12-31-43" src="https://user-images.githubusercontent.com/11621377/78356306-38229480-75a7-11ea-92e7-ee155a0eb4a5.png">

**Comment by kxxc** [comment]
> I like the Colors scan without the 3d, the reflection you used is that of a matte surface so the skeumorphism is off looking, as in, a glass CRT would gloss very differently. The border also starts being a little too prominent. Also, the color spectrum seems a bit random, try a RGBW variant maybe? I liked the first one best with the play pause I also use IINA for audio a lot and so the TV variant is not fully encompassing but since we are riffing about here. Could you try that with a rectangular background instead of a circle?
> 
> Some inspiration for tvs turning off (http://www.stephantillmans.com/luminant-point-arrays)
> 
> I'm not opposed to logo variants though it certainly is not the fashion these days
> ![screenshot 2019-03-01 17 13 27](https://user-images.githubusercontent.com/454772/53669538-61a31b00-3c45-11e9-80bb-19ec3c45fea2.png)

wow, the right one is so cool

**Comment by WoodpeckerBaby** [comment]
I think they look too TikTok...

**Comment by whatsnewsisyphus** [comment]
Playing around with a new idea ![iina-01](https://user-images.githubusercontent.com/454772/152192026-dd396067-76be-43d5-b31e-2cf5fe8d6b7c.png)
